plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -293,6 +294,7 @@ func (h *HTTPListenerV2) serveWrite(res http.ResponseWriter, req *http.Request)
 		})
 	}
 
+	queryParams := req.URL.Query()
```

Comment:
Could this be moved to within the `if h.QueryTag` check?

Reply:
Yes of course, it makes all sense. I'll move it. Thank you for the guide